recipes/faiss/build-pkg.sh

```diff
@@ -0,0 +1,17 @@
+CUDA_CONFIG_ARG=""
+if [ ${cuda_compiler_version} != "None" ]; then
+    CUDA_CONFIG_ARG="--with-cuda=${CUDA_HOME}"
+else
+    CUDA_CONFIG_ARG="--without-cuda"
+fi
+
+# Build vanilla version (no avx)
+./configure --with-blas=-lblas --with-lapack=-llapack ${CUDA_CONFIG_ARG}
+
+[[ "$target_platform" == "win-64" ]] && patch_libtool
```

Comment:
Try removing this. Looks like it doesn't use libtool.

Reply:
OK, cool. I was following https://github.com/conda-forge/autotools_clang_conda-feedstock/blob/master/recipe/meta.yaml#L55-L60, but it works without it!